src/main/java/at/hannibal2/skyhanni/features/misc/AutoJoinSkyblock.kt

```diff
@@ -0,0 +1,17 @@
+package at.hannibal2.skyhanni.features.misc
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.HypixelJoinEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.HypixelCommands
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object AutoJoinSkyblock {
+    @SubscribeEvent
+    fun onHypixelJoin(event: HypixelJoinEvent) {
+        if (!SkyHanniMod.feature.misc.autoJoinSkyblock) return
+
```

Comment:
does this not need a delay? Since hypixel does not allow commands shortly after joining a server.

Reply:
Hmm when trying it out it worked at least
Will try some more when I'm at my PC again

Reply:
And yes I thought about that too but it seemed to worked but maybe that was just luck or smth idk